compiler/main/driver.cpp

```diff
@@ -1555,8 +1553,11 @@ static void setChapelEnvs() {
       case GpuCodegenType::GPU_CG_AMD_HIP:
         gGpuSdkPath = envMap["CHPL_ROCM_PATH"];
         break;
+      case GpuCodegenType::GPU_CG_CPU:
+        gGpuSdkPath = "";
```

Comment:
Yeah, arguments you have there are the reason why I wanted to stick with it. But I did not draw parallels between this environment and `CHPL_GPU_ARCH`. That's a good point.

Strangely, though, thinking about `CHPL_GPU_ARCH` makes me want to coalesce the path variables instead of creating two different GPU arch variables. I guess I am happy with separate path variables themselves but scared of the slippery slope more.

I will create an issue for this.

Reply:
https://github.com/chapel-lang/chapel/issues/22267